db/seeds.rb

```diff
@@ -1,12 +1,27 @@
 # frozen_string_literal: true
 require "faker"
 
-lambeth = LocalAuthority.find_or_create_by!(name: "Lambeth Council", subdomain: "lambeth")
-southwark = LocalAuthority.find_or_create_by!(name: "Southwark Council", subdomain: "southwark")
+lambeth = LocalAuthority.find_or_create_by!(
+  name: "Lambeth Council",
+  subdomain: "lambeth",
+  signatory_name: "Christina Thompson",
+  signatory_job_title: "Director of Finance & Property",
+  enquiries_paragraph: "Postal address: Planning London Borough of Lambeth PO Box 734 Winchester SO23 5DG.",
```

Comment:
Is this as provided or should we have commas in the address text?

Reply:
so no one provided this, I just googled "Planning director lambeth" and popped that in...